### PR TITLE
Activate CI for query test, documentation update

### PR DESCRIPTION
**Pull request by l-hoang**
Made tests for querytest get added by default instead of conditionally.
Also updated workflow scripts and the make input call to get an updated
inputs tar that includes query test inputs. Updated the README based on
this new testing flow.

### CODE REVIEW

**Comment by l-hoang** [comment]
@gurbinder533 Note the input package is v2.1.

File: lonestar/querying/cpu/querytest/CMakeLists.txt [hunk 1]
```diff
@@ -3,17 +3,11 @@ target_link_libraries(querytest galois_query lonestar)
...
 target_compile_definitions(querytest PRIVATE USE_QUERY_GRAPH_WITH_NODE_LABEL=1)
 
-# TODO add scaling as well
-set(RESULT_CHECKER ${PROJECT_SOURCE_DIR}/scripts/result_checker.py)
 set(query_input querylist.txt)
-set(samples ${BASEINPUT}/query-tests/)
-set(query_output ${samples}/003_${query_input}.truth)
-set(ldbc_input ${BASEINPUT}/ldbc_003/meta)
+set(samples ${BASEINPUT}/cypherqueries/)
+set(ldbc_input ${BASEINPUT}/propertygraphs/ldbc_003/meta)
```
Comment (roshandathathri) [inline lonestar/querying/cpu/querytest/CMakeLists.txt:8]: Is it meta or meta_0_1? @tylershunt said it was the latter.

Comment (l-hoang) [reply lonestar/querying/cpu/querytest/CMakeLists.txt:8]: 003 is an older convert, so meta_0_1 doesn't exist for it.

It probably makes sense to reconvert 003 with the new converter, but that's also in active development, so updating it every time something gets updated on that end probably isn't scalable.

Comment (tylershunt) [reply lonestar/querying/cpu/querytest/CMakeLists.txt:8]: I'm currently fixing this so hold off on converting for now.

**Pull request merged**